### PR TITLE
#171380682 Setup empty project

### PR DESCRIPTION
**Pull request by descholar-ceo**
### What does this PR ​ do​ ? 
Setup and initialize the project

### Description of Task to be completed?

- [x] Add package.json file
- [x] Install necessary dependencies
- [x] Setup the project using webpack
- [x] Edit README.md file to include all of the necessary info about running this project
- [x] Setup test using [Jest](https://jestjs.io/docs/en/webpack)
- [x] Adding this project to TravisCI

##### A.Pre-requisites : 
* To be able to test this PR locally, you need to have the following tools on your local machine:
> * You need to have [NodeJS version between `6` higher](https://nodejs.org/en/) and [npm version `6` or higher](https://www.npmjs.com/) or [yarn v1.15.0 or higher](https://yarnpkg.com/getting-started/install)
> * Your local machine needs to be able to run terminal, Powershell, or bash commands
#### B. Steps
1. Clone this repo by running `git clone https://github.com/Stackup-Rwanda/stackup2-barefoot-frontend.git`
1. Fetch and Checkout this by running the following commands:
    > `git fetch origin ch-setup-empty-project-171380682`
    > `git checkout ch-setup-empty-project-171380682`
1. Install all needed dependencies by running `npm install` or `yarn`
1. Create a `.env` file
1. Copy all of the fields from `.env.example`, paste them in `.env` file and provide values for them
1. Start Local server by running `npm start` or `yarn start`
1. Go to `http://localhost:{{PORT}}/`   =>{{PORT}} : is the port number that you precise in your `.env` file
1. If you see this message on your browser `Hello there, welcome to the new world.
        This is the frontend of the BAREFOOT NOMAD APP developed using with
        react the-champions team!` This means this PR is working as expected.

### Any background context you want to provide?
If you are cloning this repo for the contribution purpose or adding some more feature use `npm run dev` or `yarn run dev` to enable hot reloading and browser auto-launch

### What are the relevant pivotal tracker stories?
[ #171380682](https://www.pivotaltracker.com/n/projects/2435482/stories/171380682)
[ #171380684](https://www.pivotaltracker.com/n/projects/2435482/stories/171380684)
[ #171380685](https://www.pivotaltracker.com/n/projects/2435482/stories/171380685)

### Screenshots 
N / A
### Questions:
N/A

### CODE REVIEW

**Review by Muhire-Josue — APPROVED** [review]
LGTM

**Review by the22mastermind — APPROVED** [review]
Well done @descholar-ceo 
LGTM

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#Node
```
Comment (kipropbrian) [inline .gitignore:1]: Add the .DS_Store

**Comment by kipropbrian** [comment]
I suggest we use Yarn

**Pull request merged**